packages/services/f-feature-management/src/lib/trackExperiment.js

```diff
@@ -22,7 +22,7 @@ function trackExperiment (experiment, onTrack) {
     try {
         onTrack(key, variant);
     } catch (error) {
-        logger.logError('An exception occurred while trying to send an experiment event', error);
+        logger.logError('An exception occurred while trying to send an experiment event', { error });
```

Comment:
Looks to me like `logError` [only takes one parameter](https://github.com/justeat/fozzie-components/blob/master/packages/services/f-feature-management/src/lib/logger.js#L5)?

Reply:
That's the default internal logger for non-Vue usages. See `createLoggerWrapper` for the one used when using the vue service (`FeaturesService`)